src/components/Calendar/Weekdays.tsx

```diff
@@ -9,19 +10,28 @@ export const FORMAT = 'DD/MM/YYYY';
 export interface Props {
   week?: dayjs.Dayjs[];
   onSelect?: (selected: dayjs.Dayjs) => void;
-  selected?: dayjs.Dayjs | null;
+  selected: dayjs.Dayjs[];
   state?: any;
 }
 
 function Weekdays({ week = [], onSelect = () => {}, selected, state = {} }: Props) {
+  dayjs.extend(isBetween);
+  const isSelected = (day: dayjs.Dayjs) =>
+    (selected.length > 1 &&
+      (selected[0].format(FORMAT) === day.format(FORMAT) ||
+        selected[1].format(FORMAT) === day.format(FORMAT))) ||
+    (selected!.length === 1 && selected[0].format(FORMAT) === day.format(FORMAT)) ||
+    false;
```

Comment:
lets clean this up a bit more. properly extract the format stuff to a separate function